vyper/__init__.py

```diff
@@ -2,10 +2,7 @@
 
 import pkg_resources as _pkg_resources
 
-from vyper.compiler import (  # noqa
-    compile_code,
-    compile_codes,
-)
+from vyper.compiler import compile_code, compile_codes  # noqa
```

Comment:
is `noqa` still needed?

Reply:
Yes, but it shouldn't be a blanket `# noqa` here it should be `# noqa: F401`.  Good catch.

Reply:
Fixed